test/RadworksDripsGovernance.t.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.20;
+
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ProposalTest} from "test/helpers/ProposalTest.sol";
+
+abstract contract RadworksDripsGovernance is ProposalTest {
+  function setUp() public virtual override(ProposalTest) {
+    ProposalTest.setUp();
+    _upgradeToBravoGovernor();
+  }
+
+  function _grantNewPauserViaGovernance(address _newPauser) internal {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildDripsGovernanceProposal(
+      "Grant Pauser role to an address",
+      _buildProposalData("grantPauser(address)", abi.encode(_newPauser))
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+  }
+
+  function _revokePauserViaGovernance(address _newPauser) internal {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildDripsGovernanceProposal(
+      "Revoke Pauser role from an address",
+      _buildProposalData("revokePauser(address)", abi.encode(_newPauser))
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+  }
+
+  function testFuzz_grantPauserOnDrips(address _newPauser) public {
+    assummeNotTimelock(_newPauser);
+    address[] memory _originalPausers = drips.allPausers();
+
```

Comment:
what if `_newPauser` is already in `_originalPausers`? Think that would cause this test to fail (length would not increment, correct??). Do we need to assume that `!drips.isPauser(_newPauser)`?

Reply:
Indeed.. done: https://github.com/ScopeLift/radworks-governor-upgrade/pull/18/commits/8b89ae610e34dcb41ed910f8823fe60bd52ab907